shared/version/api.go

```diff
@@ -273,6 +273,7 @@ var APIExtensions = []string{
 	"clustering_description",
 	"server_trusted_proxy",
 	"clustering_update_cert",
+	"storage_api_project",
```

Comment:
This change should be accompanied by a matching `doc/api-extensions.md` entry in the same commit.